tests/test_types.py

```diff
@@ -92,3 +92,24 @@ def test_int_in_string_raises():
     # Raises AttributeError on py2 and TypeError on py3
     with pytest.raises((TypeError, AttributeError)):
         serialize(schema, *records)
+
+
+def test_int_bytes():
+    test_cases = [(0, b"\x00"),
```

Comment:
Can you change this to use the `pytest.mark.parametrize`? An example is here:

https://github.com/fastavro/fastavro/blob/57fbe15742eb03e3a19882ebcf16b1adb2ea32dc/tests/test_logical_types.py#L196-L226